pywhat/Data/regex.json

```diff
@@ -68,7 +76,9 @@
        "URL": "https://www.blockchain.com/btc/address/",
        "Tags": [
           "Cryptocurrency",
-          "Finance"
+          "Finance",
+          "Coin Wallet",

```

Comment:
Maybe change to cryptocurrency coin wallet?

Reply:
I'm good with that, wasn't sure if there was a need for "cryptocurrency". I will make the change and fix in a bit.

Reply:
I removed `Cryptocurrency` and renamed `Coin Wallet` to `Cryptocurrency Wallet`